tools/scripts/buildModCLib.bat

```diff
@@ -40,42 +38,42 @@ goto processArgs
 
 :release
 	:: build release
-	call %~dp0\createModDir.bat CLib release
+	call "%~dp0\createModDir.bat" CLib release
 	
 	echo Building release version of CLib Mod...
 	
 	:: move away all old PBOs
-	for /f %%a IN ('dir %~dp0\..\..\PBOs\release\@CLib\addons\ /b') do move %~dp0\..\..\PBOs\release\@CLib\addons\%%a %~dp0\..\..\PBOs\archive\release\ >nul
+	for /f %%a IN ('dir "%~dp0\..\..\PBOs\release\@CLib\addons\" /b') do move "%~dp0\..\..\PBOs\release\@CLib\addons\%%a" "%~dp0\..\..\PBOs\archive\release\" >nul
 	
-	%~dp0\..\programs\armake2.exe build  %~dp0\..\..\dependencies\CLib\addons\CLib %~dp0\..\..\PBOs\release\@CLib\addons\%pboName%
+	"%~dp0\..\programs\armake2.exe" build -x pboName.h "%~dp0\..\..\dependencies\CLib\addons\CLib" "%~dp0\..\..\PBOs\release\@CLib\addons\%pboName%"

```

Comment:
Die pboName.h hatte ich absichtlich in der PBO gelassen, falls jemand die PBO entpacken will und dann mit zB mikreros Tools neu bauen will. Dann hat er auch automatisch den richtigen Namen